docs/Users-Guide.md

```diff
@@ -0,0 +1,252 @@
+## Table of Contents
+1. [Demo](#demo)
+2. [tsctl](#tsctl)
+   - [Start timesketch](#start-timesketch)
+   - [User management](#user-management)
+     - [Adding users](#adding-users)
+     - [Removing users](#removing-users)
+   - [Group Management](#group-management)
+     - [Adding groups](#adding-groups)
+     - [Removing groups](#removing-groups)
+     - [Managing group membership](#managing-group-membership)
+   - [Add index](#add_index)
+   - [Migrate database](#migrate-db)
+   - [Drop database](#drop-database)
+   - [Import JSON to timesketch](#import-json-to-timesketch)
+   - [Purge](#purge)
+   - [Search template](#search_template)
+   - [similarity_score](#similarity_score)
+3. [Concepts](#concepts)
+   - [Adding Timelines](#adding-timelines)
+   - [Using Stories](#stories)
+4. [Searching](#searching)
+
+
+## Demo
+
+To play with timesketch without any installation visit [demo.timesketch.org](https://demo.timesketch.org)
+
+## tsctl
+
+tsctl is a command line tool to control timesketch.
+
+Parameters:
+```
+--config / -c (optional)
+```
+
+Example
+```
+tsctl runserver -c /etc/timesketch.conf
+```
+
+
+### Start timesketch
+
+Will start the timesketch server
+
+Comand:
```

Comment:
Command

Reply:
Same for the rest of the file. s/Comand/Command

Reply:
ack, will change those